index.js

```diff
@@ -161,78 +155,74 @@ function repeat(n, s) {
+
+  return overrides => {
+    const callback = get ('callback', _callback, overrides);
+    const colors = get ('colors', _colors, overrides);
+    const config = get ('config', _config, overrides);
+    const leftHeader = get ('leftHeader', _leftHeader, overrides);
+    const match = get ('match', _match, overrides);
+    const rightHeader = get ('rightHeader', _rightHeader, overrides);
+    const significantDifference = get ('significantDifference',
+                                       _significantDifference,
+                                       overrides);
+
+    const table = new Table ({
       head: ['suite', leftHeader, rightHeader, 'diff', 'change', 'α'],
-      style: colors ? {} : {border: [], head: []}
+      style: colors ? {} : {border: [], head: []},
     });
 
-    var keys = (Object.keys (specs)).filter (micromatch.matcher (match));
+    const keys = (Object.keys (specs)).filter (micromatch.matcher (match));
 
     if (keys.length === 0) {
       process.stdout.write ('No benchmarks matched\n');
       callback ();
       return;
     }
 
-    var completed = 0;
+    let completed = 0;
 
     function runSpec(i) {
-      var name = keys[i];
-      var output = '# ' + String (i + 1) +
-                   '/' + String (keys.length) +
-                   ': ' +  name;
-
-      var padding = (process.stdout.getWindowSize ())[0] - output.length;
+      const name = keys[i];
+      const output = `# ${i + 1}/${keys.length}: ${name}`;
 
       process.stdout.write (
-        output + repeat (Math.max (padding, 0), ' ') + '\r'
+        `${output.padEnd ((process.stdout.getWindowSize ())[0])}\r`
       );
 
-      var suite = new Suite (name);
-      var spec = specs[name];
-      var left = spec[1];
-      var right = spec[spec.length - 1];
+      const suite = new Suite (name);
+      const spec = specs[name];
+      const left = spec[1];
+      const right = spec[spec.length - 1];
 
       suite.add ('left', Object.assign ({}, config, spec[0], {
-        fn: function() { left (leftLib, arguments); }
+        fn: (...args) => { left (leftLib, args); },
       }));
 
       suite.add ('right', Object.assign ({}, config, spec[0], {
-        fn: function() { right (rightLib, arguments); }
+        fn: (...args) => { right (rightLib, args); },
       }));
 
       suite.on ('complete', function() {
-        var oldRes = this[0], newRes = this[1];
-        var change = (newRes.hz - oldRes.hz) / oldRes.hz;
-        var difference = Math.abs ((newRes.hz - oldRes.hz) /
-                                   ((oldRes.hz + newRes.hz) / 2) /
-                                   2);
-        var isPositive = difference > significantDifference && change > 0;
-        var isNegative = difference > significantDifference && change < 0;
-
-        var highlight = colors && isPositive ? green :
-                        colors && isNegative ? red :
-                        identity;
+        const {0: oldRes, 1: newRes} = this;
```

Comment:
I spent significant time debugging an error that resulted from trying `[oldRes, newRes]` here.